sites/fast-color-explorer/app/site-footer.tsx

```diff
@@ -13,7 +13,7 @@ export class Footer extends React.Component<{}, {}> {
             <fast-design-system-provider
                 class="site-wrapper"
                 use-defaults
-                dangerouslySetInnerHTML={{ __html: FooterTemplate }}
+                dangerouslySetInnerHTML={{ __html: FooterTemplate() }}
```

Comment:
Is this related to the windows validation issues?

Reply:
This also for windows validation. John have more understanding of this than me.

Reply:
@chrisdholt The package `rollup-plugin-ejs` has `node-sass` as a dependency, and that fails to install properly on Windows environments. The `rollup-plugin-jst` package provides the functionality we need without that dependency, but the calls here need to be referenced as functions instead of constants.